pkg/worker/lifecycle.go

```diff
@@ -175,7 +177,7 @@ func (s *Worker) RunContainer(request *types.ContainerRequest) error {
 	}
 	log.Printf("<%s> - acquired port: %d\n", containerId, bindPort)
 
-	// Read spec from bundle
+	// Read spec from bundle. If not found, then this is a container being built on a build worker.
```

Comment:
It seemed like it was always true from my testing. Seemed to make sense since we were not doing anything with the error too. 

But either way, I'm not doing anything here any more, so I'll just revert the comment.